src/solvers/smt2_incremental/type_size_mapping.h

```diff
@@ -6,13 +6,12 @@
 #ifndef CPROVER_SOLVERS_SMT2_INCREMENTAL_TYPE_SIZE_MAPPING_H
 #define CPROVER_SOLVERS_SMT2_INCREMENTAL_TYPE_SIZE_MAPPING_H
 
-#include <util/expr.h>
+#include <util/expr.h> // IWYU pragma: keep
 
-#include <solvers/smt2_incremental/ast/smt_terms.h>
```

Comment:
`smt_termt`is defined in `smt_terms.h` and is used in the definition of `type_size_mapt` in this file. Therefore this include should be kept.

Reply:
I have now added `// IWYU pragma: keep` instead.